assert/values.go

```diff
@@ -155,9 +159,13 @@ func (f FixtureFile) AssertResponseBody(t *testing.T, requestInfo string, respon
 
 	//write actual content to file to make it easy to copy the computed result over
 	//to the fixture path when a new test is added or an existing one is modified
-	fixturePathAbs, _ := filepath.Abs(string(f))
+	fixturePathAbs, err := filepath.Abs(string(f))
+	if err != nil {
+		t.Fatal(err)
+		return false
+	}
 	actualPathAbs := fixturePathAbs + ".actual"
-	err := os.WriteFile(actualPathAbs, responseBody, 0644)
+	err = os.WriteFile(actualPathAbs, responseBody, 0644) //nolint:gosec // only used for tests
```

Comment:
As discussed, please disable this lint. And actually...
```suggestion
	err = os.WriteFile(actualPathAbs, responseBody, 0666)
```